custom-recipes/nlp-analysis-ontology-tagging/recipe.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+from dataiku.customrecipe import *
+import dataiku
+import pandas as pd
+from dataiku import pandasutils as pdu
+from dataiku.customrecipe import get_recipe_config
+from tagger import Tagger
+
+#All plugin parameters name
+MANDATORY_PARAMS = ['text_column',
+                    'language_column',
+                    'tag_column',
+                    'case_sensitivity',
+                    'lemmatization',
+                    'unicode_normalization',
+                    'possible_outputs']
+
+OPTIONAL_PARAMS  = ['keyword_column',
+                   'category_column']
+
+
+#raise an error when called
+class ParameterError(ValueError):
+    pass
+
+#get input datasets
+def get_input_datasets(role):
+    return dataiku.Dataset(get_input_names_for_role(role)[0])
+
+def get_output_datasets(role):
+    return dataiku.Dataset(get_output_names_for_role(role)[0])
+
+def load_settings():
```

Comment:
Can you use Henri's library for loading parameters from the frontend to the backend?
https://github.com/dataiku/dss-plugin-dkulib/tree/main/dkulib/dku_config

Reply:
done in commit 16

Reply:
Getting there! Can you move this to a separate module within python-lib? For instance, something called "dku_plugin_config_loading"

Reply:
done : in cc8c943 (recipe.py) and d2e542c